homeassistant/components/lock/services.yaml

```diff
@@ -1,3 +1,25 @@
+clear_usercode:
+  description: Clear a usercode from lock

```

Comment:
Looks like this file has come initially from a windows machine containing CRLF instead of just LF.